channels/unsorted.m3u

```diff
@@ -5732,3 +5732,11 @@ https://5cf58a556f9b2.streamlock.net:443/live/tv_jejumbc/playlist.m3u8
 https://5c74939c891dc.streamlock.net/live/TV/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="http://www.etv.go.kr/images/newimg/logo.png" group-title="",한국선거방송
 http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index_1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/locomotionxtv/picture?width=300&height=300" group-title="",Locomotion TV 1
+http://locomotiontv.com/envivo/loco_ch/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/locomotionxtv/picture?width=300&height=300" group-title="",Locomotion TV 2
+http://51.222.85.85:81/hls/loco/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/AnimeZoneXTV/picture?width=300&height=300" group-title="",AZXTV
+http://azxtv.com/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/limtvok/picture?width=300&height=300" group-title="",LIM TV
+https://live.admefy.com/live/default/ashamed_crimson_3360d.m3u8
```

Comment:
Not 24/7.

Reply:
As far as I know, the Locomotion and AZXTV version is operational in the evenings (Argentina time), although its programming is repeated in the early mornings and mornings.

La Invasiva TV was operative yesterday, the website mentions their schedule: https://lainvasiva.com/index.php/programacion/, but they clarified that there was a computer attack that affected their operations.

Reply:
The second Locomotion link works for me.

AZX also works. I'm saying that it should be tagged as such.

To do so, just add [Not 24/7] at the end of the names.

Reply:
You mean add the [Not 24/7] tag for the three channels. Right?

Reply:
Ok, I going to edit